docs/integrations/spark.md

```diff
@@ -55,42 +51,31 @@ interact with your data on lakeFS.
 
   </div>
   <div markdown="2" id="install-databricks">
+In  your cluster settings, under the _Libraries_ tab, add the following Maven package:
 
-**Note** For your convenience, follow this [step by step Databricks integration tutorial, including lakeFS Hadoop File System, Python client and lakeFS SPARK client](https://lakefs.io/blog/databricks-lakefs-integration-tutorial/).
-{: .note }
-
-
-  Add the `hadoop-lakefs-assembly` jar to your Databricks cluster. Under `compute ➡️ Libraries` Add a Maven package as follows:
-
-  Install a Maven package with the `io.lakefs:hadoop-lakefs-assembly:0.1.12` 
-  coordinates.
-
-  Once installed, it should look something like this:
+```
+io.lakefs:hadoop-lakefs-assembly:0.1.12
+```
 
-  ![Databricks - Adding the lakeFS client Jar](../assets/img/databricks-install-package.png)
+Once installed, it should look something like this:
 
+![Databricks - Adding the lakeFS client Jar](../assets/img/databricks-install-package.png)
   
   </div>
 </div>
 
 
 ### Configuration
 
-Configure the S3A filesystem with your S3 credentials (**not** the lakeFS credentials).
-Additionally, supply the `fs.lakefs.*` configurations to allow Spark to access metadata on lakeFS: 
+Set the `fs.lakefs.*` Hadoop configurations to point to your lakeFS installation:
+* `fs.lakefs.impl`: `io.lakefs.LakeFSFileSystem`
+* `fs.lakefs.access.key`: lakeFS access key
+* `fs.lakefs.secret.key`: lakeFS secret key
```

Comment:
I liked the emphasis "**not** the lakeFS credentials", I think it covers a fair number of configuration issues.  Can we keep something like this?